### PR TITLE
Print a message instead of logging into Rack after install

### PR DESCRIPTION
**Pull request by soulshake**
Addresses #2022

### CODE REVIEW

**Comment by soulshake** [comment]
[Print a message instead of logging into Rack after install](https://trello.com/c/u4UomxQV/2150-print-a-message-instead-of-logging-into-rack-after-install)

File: cmd/convox/install.go [hunk 1]
```diff
@@ -443,27 +443,27 @@ func cmdInstall(c *cli.Context) error {
...
 
 	return stdcli.QOSEventSend("cli-install", distinctID, ep)
 }
 
+func addRackToConsoleMsg(c *cli.Context, host string, password string) {
+	fmt.Println("** Success! **")
+	fmt.Println("Your Rack has been installed. You should now add it as an existing Rack at console.convox.com (or your own private Console) with the following credentials (which have also been written to ~/.convox/auth):")
+	fmt.Printf(" 🢂  Hostname: %s\n", host)
+	fmt.Printf(" 🢂   API Key: %s\n", password)
+	fmt.Println("\nAdvanced: To log into this Rack directly instead, run:")
```
Comment (ddollar) [inline cmd/convox/install.go]: Probably don't even need the direct instructions here.

**Comment by MiguelMoll** [comment]
This PR should change CI steps to login after successful install.

**Comment by mattmanning** [comment]
[update docs to reflect new post-install behavior for CLI Rack installs](https://trello.com/c/SgPvmGVt/2301-update-docs-to-reflect-new-post-install-behavior-for-cli-rack-installs)

**Pull request closed without merging**